doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -51,10 +71,19 @@
 
 > This chapter should describe considerations about CPU management.
 
+> Relate back to features described in the RM [here](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter05.md#521-virtual-compute) and [here](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter05.md#54-nfvi-hw-profiles-features-and-requirements).
+
+
 <a name="3.2.1.5"></a>
 #### 3.2.1.5 Container Runtime Services
 
-> This chapter should describe considerations about the services of container runtimes.
+The Container Runtime is the component that runs within a Host Operating System (OS) and manages the underlying OS functionality, such as cgroups and namespaces (in Linux), in order to provide a service within which container images can be launched. For the purposes of this document the Container Runtime is not just the true 'runtime' but also the container 'engine' that manages the container-related infrastructure such as networking, security, storage and distributed state.  Essentially the component that pulls an image from a registry, unpacks it and runs it.
```

Comment:
in a K8s infrastructure the container runtime typically does not manage the containers networking, and storage needs. also not much of the security aspects

Reply:
Ok, I will look to re-word, as the runtime does absolutely manage those resources - e.g. network namespace in the OS, access to devices, IO throttling etc. But as I suspect you mean, it is always on instruction from Kubernetes so I'll try and clarify that.

Reply:
OK - reworded in latest commit